pkg/controller/constants/constants.go

```diff
@@ -73,7 +73,8 @@ const (
 	NativeImageMarker         = "native"
 	GeneratedSecretSuffix     = "generated-secret"
 	InfinispanFinalizer       = "finalizer.infinispan.org"
-	SiteServiceTemplate       = "%v-site"
+	SiteServiceNameTemplate   = "%v-site"
+	SiteServiceFQNTemplate    = "%s.%s.svc.cluster.local"
```

Comment:
Does this need to be a constant? It's only used once.

Reply:
Yes, exactly. I will move both `SiteServiceNameTemplate` and `SiteServiceFQNTemplate` to the `types_util.go`